charts/quickwit/templates/prometheusrule.yaml

```diff
@@ -5,9 +5,12 @@ metadata:
   name: {{ include "quickwit.fullname" . }}
   labels:
     {{- include "quickwit.labels" . | nindent 4 }}
+    {{- with .Values.prometheusRule.additionalLabels }}
```

Comment:
can we add `prometheusRule.additionalLabels` to the `values.yaml` ?

Reply:
oh i forget that. done.